app/models/event/tour.rb

```diff
@@ -20,6 +20,7 @@ class Event::Tour < Event
   ]
 
   self.supports_applications = true
+  self.supports_invitations = false
```

Comment:
This does not really belong to this feature, if that is needed for sac tours, you can keep it in tho :upside_down_face: